src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, *args, **kwargs):
 
     @skipForTicket("DAOS-3142")
     def test_no_space_cont_create(self):
+        # pylint: disable=no-member
```

Comment:
I'd suggest using the `general_utils.py` `get_random_string()` method to remove the `sting.lowercase` no-member warning